style1.css

```diff
@@ -0,0 +1,418 @@
+
```

Comment:
It would be more reasonable to use flex for all classes. Because always to write display: flex is not so good. 
Be attentive to the styling. I need to read all code and it's difficult to review if there is no consistancy in the code writing